salesagility/SuiteCRM
# Fix/email css error

## Description



Small CSS & JS Change to resolve miscoloured field background visible when creating a personal inbound email

## Motivation and Context

Looks nasty + HacktoberFest + Trick or Suite Thursdays! :smile: 

## How To Test This


1. Navigate to profile
2. Scroll to the bottom of the page and hit settings
3. Press Mail Accounts
4. See that when no record is provided there is a purple box under mail accounts
5. Replicate with fix installed and see white box

## Types of changes


### Final checklist
